### ISSUE TITLE
Tree table jumps

### ISSUE THREAD

**Issue by sezmars** [primefaces/primeng#7527]
```
[x] bug report => Search github for a similar issue or PR before submitting
```
Hi guys @cagataycivici, @vincent @yigitfindikli
-
I have the same problem as on your site, watch the video

https://drive.google.com/file/d/1kQw1kMKoXHhRqbT8UnBcR_zYTBkWOjwa/view

>Windows Pro 10 x64
Angular version: 7.2.9
PrimeNG version: 7.1.0
Browser: Chrome - 73.0.3683.103

**Comment by yigitfindikli** [comment]
Hi,
Issue is resolved now for 7.1.1
You can see in https://github.com/primefaces/primeng/commit/f4863691ce98a568561ac5837a886d8957857770 .

**Comment by sezmars** [comment]
> Hi,
> Issue is resolved now for 7.1.1
> You can see in [f486369](https://github.com/primefaces/primeng/commit/f4863691ce98a568561ac5837a886d8957857770) .

Not working and no matching version found for primeng@^7.1.1.

**Comment by yigitfindikli** [comment]
Next release is 7.1.1, you can see in milestones.
